pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: github
-version: 9.0.1
+version: 9.0.2-dev
```

Comment:
same here, straight to 9.0.2

Reply:
Zero effect on the shipping product. No rush.